pkg/forklift-api/webhooks/validating-webhook/admitters/provider-admitter.go

```diff
@@ -84,6 +83,19 @@ func (admitter *ProviderAdmitter) validateVDDK() error {
 	return nil
 }
 
+func (admitter *ProviderAdmitter) validateVddkImage() error {
+	if image, found := admitter.provider.Spec.Settings[api.VDDK]; found {
+		if image == "" {
+			err := liberr.New("The specified VDDK init image is empty")
+			log.Error(err, "empty image is invalid, failing",
```

Comment:
if the above it true, i would suggest to say only omitted or image value is accepted to hint what is it "empty"